taskmanager_app/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe 'タスク管理機能', type: :system do
+  let!(:task_a) { FactoryBot.create(:task, name: '最初のタスク') }
+
+  describe '一覧表示機能' do
+    it '一覧表示のページで最初のタスクが表示される' do
+      visit(tasks_path)
+      expect(page).to have_content('最初のタスク')
+    end
+  end
+
+  describe '詳細表示機能' do
+    it '最初のタスクの詳細が表示される' do
+      visit(tasks_path(task_a))
```

Comment:
```suggestion
      visit(task_path(task_a))
```
後々説明文のテストを足した時にURLを間違えてたことに気づきました
(後の方のコミットで修正してます)